web/models/attendance/attendance.ex

```diff
@@ -14,9 +17,32 @@ defmodule CoursePlanner.Attendance do
 
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:attendance_type, :class_id, :student_id])
+    |> cast(params, [:attendance_type, :class_id, :student_id, :comment])
     |> validate_required([:attendance_type, :class_id, :student_id])
     |> cast_assoc(:class)
     |> cast_assoc(:student)
+    |> validate_comment()
```

Comment:
you may extract `Settings.get_value("ATTENDANCE_DESCRIPTIONS")` to a function in order to make it nicer, but we shouldn't reinvent Ecto validations